Tests/IntegrationTests.Shared/PerformanceTests.cs

```diff
@@ -133,6 +133,37 @@ public void BindingSetValuePerformanceTest(int count)
             Console.WriteLine("Kilo-iterations per second: {0:0.00}", ((count/1000) / sw.Elapsed.TotalSeconds));
         }
 
+        string benchmarkString;
+
+        [Test, Explicit("Temporary benchmarker")]
```

Comment:
How about a quick comment saying why this is here and what it was being used to compare against?